### PR TITLE
New Project Idea

### PR DESCRIPTION
**Pull request by AnashaSarker2001**
We have changed our project idea due to difficulties and lack of resources. @platisd

### CODE REVIEW

**Review by SadhanaAnandan — APPROVED** [review]
This looks good.

File: README.md [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
 
-Obtacles: pedestrians, cones, construction barricades, police car crossing the crowd.
 
-Boosters: speed up, barrier, additional time, extra life.
+# HOW?
+ARDUINO is going to be used to code the software for the car
+ANDROID STUDIO to build and design the app
+MQTT to connect the app to the emulator
+SMCE is where the car itself and all its features will be implemented
+GODOT to modify the environment and add obstacles
```
Comment (platisd) [inline README.md:16]: Why is GODOT still here?

File: README.md [hunk 2]
```diff
@@ -1,17 +1,23 @@
 # group-11
 
-Imagine you rob a bank, the plan goes perfectly and nobody manages to stop you while you exit the building with all the money you just proudly stole. BUT you missed one small detail of the robbery plan... THE GETAWAY! You did not think about how you're going to escape town with your car at all and the police is everywhere. Dont even get me started on the civilians being all over the place! You must now use your driving skills and prove worthy of all the money you just got and the title of the best robber of all time.
 
+# What 
+We are creating an application to drive the car maunally for seeking treasure(s). The car will be able to take multiple routes to find the treasure. The environment will consist of multiple objects/obstacles and one treasure. The user will be alerted when there is an object nearby and the car will be stopped if it gets too close to the object. The closer or further the car is from the treasure will be displayed on the app. The user will be able to view the car's speed and how much the car has travelled so far. The user will have a first person view (driver seat view).
```
Comment (platisd) [inline README.md:5]: What will be the treasure?
How will the treasures be "detected"? (with or without quotes)

File: README.md [hunk 3]
```diff
@@ -1,17 +1,23 @@
...
-Boosters: speed up, barrier, additional time, extra life.
+# HOW?
+ARDUINO is going to be used to code the software for the car
+ANDROID STUDIO to build and design the app
+MQTT to connect the app to the emulator
+SMCE is where the car itself and all its features will be implemented
+GODOT to modify the environment and add obstacles
+
+# EXTRA (if we have extra time)
+- Rear camera view
```
Comment (platisd) [inline README.md:19]: A camera view is not difficult, it shouldn't be listed as an extra.

File: README.md [hunk 4]
```diff
@@ -1,17 +1,23 @@
...
+# HOW?
+ARDUINO is going to be used to code the software for the car
+ANDROID STUDIO to build and design the app
+MQTT to connect the app to the emulator
+SMCE is where the car itself and all its features will be implemented
+GODOT to modify the environment and add obstacles
+
+# EXTRA (if we have extra time)
+- Rear camera view 
+- Random positions of the hidden object
```
Comment (platisd) [inline README.md:20]: Refer to the GODOT question.

File: README.md (hunk 1 above)
Comment (shahdmetwally) [reply README.md:16]: We need Godot to add the objects to the environment that is already existing in the SMCE and modify them.

File: README.md (hunk 3 above)
Comment (shahdmetwally) [reply README.md:19]: Okay. Thanks, we will move the rear view from later development.

File: README.md (hunk 2 above)
Comment (shahdmetwally) [reply README.md:5]: - The treasure will be an object different from all the other objects.
- The treasure will be detected by moving in place but the other objects will be still, therefore we will be using an infrared sensor.

File: README.md (hunk 1 above)
Comment (platisd) [reply README.md:16]: Didn't we agree that you will **not** be going for modding at all?

**Pull request merged**